src/app/shared/unitOptions.ts

```diff
@@ -83,8 +83,8 @@ export const VolumeGasOptions: Array<UnitOption> = [
         unitsOfMeasure: 'Imperial'
     },
     {
-        display: 'Normal Meter Cubed (m&#x00B3;)',
-        value: 'm3',
+        display: 'Normal Meters Cubed (Nm&#x00B3;)',
+        value: 'Nm3',
```

Comment:
We still want "m3" as the value because that is what the definition in the unit converter will use, it shouldn't ever show in display